sdk/core/azure-core/src/etag.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/core/etag.hpp"
+
+using Azure::ETag;
+
+const ETag& ETag::Any()
+{
+  static ETag any = ETag("*"); // LCOV_EXCL_LINE
```

Comment:
> LCOV_EXCL_LINE

What's this and why do we need it?

Reply:
Thanks! This may be added later, as part of a different PR that addresses code coverage. I missed the removal of this comment for this PR.

Reply:
```suggestion
  static ETag any = ETag("*");
```